lib/MusicBrainz/Server/Entity/URL/CPDL.pm

```diff
@@ -0,0 +1,36 @@
+package MusicBrainz::Server::Entity::URL::CPDL;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name {
+    my $self = shift;
+
+    return "Score at CPDL";
```

Comment:
Maybe that should be labelled more explicitly as "Choral Public Domain Library". @reosarevok: Any opinion?